build.sbt

```diff
@@ -1303,9 +1304,9 @@ lazy val examplesTomcat = exampleProject("examples-tomcat")
 
 lazy val scalafixInternalRules = project
   .in(file("scalafix-internal/rules"))
-  .enablePlugins(NoPublishPlugin)
   .disablePlugins(ScalafixPlugin)
   .settings(
+    name := "http4s-scalafix-internal",
```

Comment:
Hoping that it's as simple as publishing them. We shouldn't guarantee bincompat though. Bikeshed the name.